src/net.gr

```diff
@@ -64,8 +64,8 @@ provide module DNS {
     ) {
       let address = match (WasmI32.load(ipPointer, _TYPE_OFFSET)) {
         4n => {
-          let (<<) = WasmI32.shl
-          let (|) = WasmI32.or
+          let (<<) = WasmI32.(<<)
```

Comment:
A note on these is this should really use the `from` syntax as it gets inlined vs the let syntax which will not.